uk.co.compucorp.civicrm.hrleaveandabsences/tests/phpunit/api/v3/LeaveRequestTest.php

```diff
@@ -4366,6 +4367,61 @@ public function testTheTimeForFromAndToDateOfLeaveRequestIsAddedCorrectlyWhenLea
     $this->assertEquals($toDate->format('Y-m-d') ." 23:59:00", $leaveRequest->to_date);
   }
 
+  public function testTheTimeForFromAndToDateOfLeaveRequestStaysUntouchedWhenRequestIsTOIL() {
+    $absenceType = AbsenceTypeFabricator::fabricate([
+      'allow_accruals_request'   => true,
+      'max_leave_accrual'        => 10,
```

Comment:
Please leave a single space between the array key operand and value i.e` 'max_leave_accrual' => 10,` for this array.

Reply:
Sure. I just followed this from other places in the same file.

Reply:
Done.